chat/README.md

```diff
@@ -0,0 +1,71 @@
+# Fine-tuning StarCoder for chat-based applications
+
+This is an educational example to fine-tune `StarCoderBase` on a corpus of multi-turn dialogues and thus create a coding assistant that is chatty and helpful. Check out our [blog post](ADD LINK) for more details.
+
```

Comment:
maybe some examples of what the model can do after tuning would be cool to show here.